streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -667,8 +668,11 @@ public void run() {
      * {@link TopologyBuilder} and {@link KStreamBuilder}).
      *
      * @return A string representation of the Kafka Streams instance.
+     *
+     * @deprecated Use {@link #localThreadsMetadata()} to retrieve runtime information.
```

Comment:
We should mention it gets deprecated by two functions:

1) to get the static full topology description, one should use `Topology#describe()`;
2) to get the runtime thread metadata, one should use `#localThreadsMetadata()`;